Langs/0407.lang

```diff
@@ -70,13 +70,13 @@ Loading WMIC=Lade WMIC
+CPU Name=Die Prozessor, der in Ihrem Computer verbaut ist. Die Windows 11 Kompatibilität kann sich noch ändern. Bei einer Inkompatibilität muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
+CPU Cores=Die Anzahl der Rechenkerne, die Ihrer CPU zur Verfügung stehen. Bei einer Inkompatibilität muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
+CPU Speed=Die Geschwindigkeit, mit der Ihre CPU-Aufgaben verarbeitet. Bei Inkompatibilität muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
 DirectX=Die Version des DirectX-Funktionslevels, den die Grafikkarte unterstützt. Dies ist unabhängig von der DirectX-Softwareversion. Es kann vorkommen, dass 'DirectX 12 API'-Karten diese Prüfung nicht bestehen.
-Disk Type=Der Partitionsstil, in dem Ihre Daten auf Ihrer Festplatte oder SSD gespeichert sind. Windows 11 benötigt GPT. Sollten Sie MBR anstatt GPT nutzen, können Sie dies mit dem MBR2GPT-Tool von Microsoft korrigieren.
-RAM=Die Größe des in Ihrem Computer installierten Arbeitsspeichers. Erweitern oder wechseln bei Desktop PCs sowie High-End-Laptops meistens möglich.
-Secure Boot=Eine Methode, die Ihr Motherboard zum Booten von Windows verwendet. Wenn nicht erkannt, kann dies auf neueren Motherboards den BIOS/UEFI-Einstellungen behoben werden. Schlagen Sie im Handbuch Ihres Motherboards nach.
-Storage=Der Speicherplatz Ihrer Festplatte oder SSD. Eine Aufrüstung ist bei Desktop PCs sowie High-End-Laptops meistens möglich.
-TPM=Ein Sicherheitsmodul, das von Windows verwendet wird. Nicht immer ist ein separates Hardwaremodul notwendig. Viele moderne CPUs enthalten diese Funktion. AMD nennt es "fTPM", Intel "PTT". Überprüfen und aktivieren Sie diese Funktion in den BIOS/UEFI-Einstellungen. Schlagen Sie im Handbuch Ihres Motherboards nach.
+Disk Type=Der Partitionsstil Ihrer Festplatte oder SSD. Windows 11 benötigt eine GPT-Partition. Sollten Sie MBR anstatt GPT nutzen, können Sie dies mit dem MBR2GPT-Tool von Microsoft korrigieren.
+RAM=Die Größe des in Ihrem Computer installierten Arbeitsspeichers. Eine Aufrüstung ist bei Desktop PCs sowie vielen Laptops meistens möglich.
+Secure Boot=Secure Boot (sicherer Start) ist eine Funktion innerhalb des UEFI-BIOS. Für Windows 11 muss diese Funktion vorhanden sein. Wenn Secure Boot nicht erkannt wird, kann dies bei neueren PCs oder Laptops im UEFI-BIOS aktiviert werden. Schlagen Sie im Handbuch Ihres Motherboards nach.
+Storage=Der Speicherplatz Ihrer Festplatte oder SSD. Eine Aufrüstung ist bei Desktop PCs sowie vielen Laptops meistens möglich.
+TPM=Trusted Platform Module (TPM) ist ein Sicherheitsmodul des Computers, dass von Windows 11 benötigt wird. Nicht immer ist ein separates TPM-Modul notwendig. Viele moderne CPUs enthalten diese Funktion. Die Aktivierung von TPM ist oft in den UEFI-BIOS-Einstellungen möglich. Im BIOS nennt sich diese Funktion "fTPM" (AMD) oder "PTT" (Intel). Schlagen Sie im Handbuch Ihres Motherboards nach.
```

Comment:
```suggestion
TPM=Trusted Platform Module (TPM) ist ein Sicherheitsmodul des Computers, das von Windows 11 benötigt wird. Nicht immer ist ein separates TPM-Modul notwendig. Viele moderne CPUs enthalten diese Funktion. Die Aktivierung von TPM ist oft in den UEFI-BIOS-Einstellungen möglich. Im BIOS nennt sich diese Funktion "fTPM" (AMD) oder "PTT" (Intel). Schlagen Sie im Handbuch Ihres Motherboards nach.
```

Reply:
I think `dass`  is correct here or not?

Reply:
If it could be replaced with `wepches`, you need `das`!

Reply:
> If it could be replaced with `welches`, you need `das`!

😅 Ups

Reply:
Should this change also be commited or not?